index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" href="style.css">
+
+  <script src="script.js"></script>
+
+  <title>Tic Tac Toe</title>
+
+</head>
+
+<body onload="startGame();">
```

Comment:
Why not move the `js` to its own file? Instead of using in-line function calls and click handlers